sequencer/src/options.rs

```diff
@@ -277,6 +277,7 @@ module!("query", api::options::Query, requires: "http");
 module!("submit", api::options::Submit, requires: "http");
 module!("status", api::options::Status, requires: "http");
 module!("state", api::options::State, requires: "http");
```

Comment:
Maybe add "sql-storage" to the requires list, since this module is ignored if we have file system storage